auth-web/src/components/auth/NonBcscAccounts/AffidavitDownload.vue

```diff
@@ -73,37 +73,64 @@
         Back
       </v-btn>
       <v-spacer></v-spacer>
-      <v-btn
-        large
-        color="primary"
-        class="font-weight-bold mr-2"
-        data-test="use-existing-bceid-btn"
-        @click="signinUsingBCeID"
-      >
-        Login using existing BCeID
-      </v-btn>
-      <v-btn
-        large
-        color="primary"
-        class="font-weight-bold"
-        data-test="register-bceid-btn"
-        @click="redirectToBceId"
-      >
-        Register a new BCeID
-      </v-btn>
+      <template v-if="isAuthenticated">
```

Comment:
is this change related to this ticket?

Reply:
the changes looks like if isAuthenticated , then show the signin buttons ? a bit confused here..could be a correct change , just wants to understand it more

Reply:
Yes, its related to this.
actually if isAuthenticated, then we show the Continue button, otherwise the signin/register buttons